atlas/main_test.go

```diff
@@ -62,7 +64,10 @@ func TestMain(m *testing.M) {
 	log.Print("wait for servers to load up")
 	time.Sleep(time.Second)
 
-	m.Run()
+	code := m.Run()
+	// os.Exit() does not respect defer statements
```

Comment:
this line can be removed

Reply:
You're suggesting removing the comment? The comment is there so that future maintainers of the code will know they cannot safely use `defer`. I don't think most of the team is familiar with this limitation.

Reply:
makes sense, lets leave it as is then